src/engine/sparqlExpressions/NowDatetimeExpression.h

```diff
@@ -0,0 +1,52 @@
+//  Copyright 2024, University of Freiburg,
+//                  Chair of Algorithms and Data Structures
+//  Author: Hannes Baumann <baumannh@informatik.uni-freiburg.de>
+
+#include "engine/sparqlExpressions/SparqlExpression.h"
+#include "util/ChunkedForLoop.h"
+#include "util/Date.h"
+#include "util/Random.h"
+
+namespace sparqlExpression {
+
+// The expression NOW() is evaluated within NowDatetimeExpression.
+// NowDatetimeExpression has to be explicitly instantiated with a
+// date-formatted string, which is for all evaluations within a Sparql query the
```

Comment:
`constructed from`  not `instantiated with`, the latter verb is used only for template instantiations.